packages/wrangler/src/config/environment.ts

```diff
@@ -374,6 +374,12 @@ interface EnvironmentNonInheritable {
 
 			/** The queue to send messages that failed to be consumed. */
 			dead_letter_queue?: string;
+
+			/** Whether the Queue broker will make concurrent consumer invocations */
+			concurrency_enabled?: boolean;
+
+			/** The maximum number of concurrent consumer Worker executions */
+			max_concurrency?: number;
```

Comment:
What happens if you put a number larger than we support? e.g. 21? Or 0, or -1 - ?

Reply:
It will try to send the invalid value to the API, and fail (with a reasonable error message). That is in line with the other settings at the moment. 

I suppose we could add more client side validation to all of the queue settings, but it is a little annoying to maintain them in two places.

Reply:
I don't think we need more client validation, but:

- What does 0 do? Is it invalid or coerced to null?
- Is -1 in invalid?
- What is the current max?

(These will go into the docs)

Reply:
Ah right, so:
* 0 is invalid
* -1 is invalid
* 20 is the current max enforced by the backend. Shall we drop it to 5?

So to summarize, the user's value must be between 1 and the max, inclusive.
They can also omit the value / specify `null` to indicate that they don't care to enforce a limit (we'll use the highest possible value behind the scenes)

Reply:
Meet in the middle at 10 for now?

Rest makes sense - thanks!